_summaries/chen2020a_1.md

```diff
@@ -1,19 +1,40 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: {chen2020a_1}
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: specfazhou
+score: 8/10
 ---
 
 TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
+* What is the core idea? <br/>
+This paper comes up with a simple framework for contrasive learning of visual representation, which not only archieves state of the art accuracy, but also has simplier structure. The result of SimCLR is shown in the following picture <br/>
```

Comment:
Since you bring up that this framework is simpler, I think it could be good to mention somewhere what complicated things previous works were doing.

Reply:
I think during the presentation, other students will say previous works first, maybe I just leave it like this

Reply:
added the fact that previous works require memory bank and specialized architectures